hdmi2usb/tests/dev_setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -x
+
+opsis-off.sh
+atlys-off.sh
+
+sleep 3
+opsis-on.sh
+sleep 3
+
+atlys-on.sh
+sleep 3
+
+(cd ~/prebuilt
+~/HDMI2USB-mode-switch/venv/bin/hdmi2usb-mode-switch -v \
+    --by-type atlys \
+    --load-fx2-firmware hdmi2usb.hex
+)
+
+sleep 3
+sudo ln -sf /dev/video0 /dev/hdmi2usb/by-num/opsis0/video
+
+sudo ln -sf /dev/ttyVIZ0 /dev/hdmi2usb/by-num/atlys0/tty
+sudo ln -sf /dev/video2 /dev/hdmi2usb/by-num/atlys0/video
```

Comment:
I doubt it will always be `/dev/video2`? Also, where is `/dev/video1`?

Reply:
This is to work around the udev issues.  Once they are working as expected this can all go away.
```
pi@oppi:/dev $ ls video?
video0  video1  video2  video3

/dev/hdmi2usb/by-num/atlys0/video -> ../../../video1
```
no clue about video3.  I'm assuming it will go away too.